tests/common.go

```diff
@@ -6161,7 +6162,13 @@ func CollectLogsFromPods(testCaseName string, podLabel map[string]string, namesp
 
 	// Check to handle cloud based deployment with 0 master nodes
 	if len(node.GetMasterNodes()) == 0 {
-		log.Warnf("Skipping pod log collection for pods with [%s] label in test case [%s]", logLabel, testCaseName)
+		log.Warnf("Skipping pod log collection for pods with [%s] label in test case [%s] as it's cloud cluster", logLabel, testCaseName)
+		return
+	}
+
+	// In case of ocp skip log collection
+	if Inst().S.String() != openshift.SchedName {
```

Comment:
== ?

Reply:
I think it should be ==

Reply:
Thanks for pointing out saw that was fixing it.